report_xml/README.rst

```diff
@@ -0,0 +1,102 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Qweb XML Reports
```

Comment:
put `================` on top of the title
the proper name of the module is `Report XML`